### PR TITLE
Fix documentation of `range` to reflect the truth.

### PR DESCRIPTION
**Pull request by sherfert**

### CODE REVIEW

File: cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ListFunctionsTest.scala [hunk 1]
```diff
@@ -99,13 +99,17 @@ class ListFunctionsTest extends DocumentingTest {
     section("range()", "functions-range") {
       p(
         """`range()` returns a list comprising all integer values within a range bounded by a start value `start` and end value `end`, where the difference `step` between any two consecutive values is constant; i.e. an arithmetic progression.
-          |The range is inclusive, and the arithmetic progression will therefore always contain `start` and -- depending on the values of `start`, `step` and `end` -- `end`.""".stripMargin)
+          |To create ranges with decreasing integerer values, use a negative value `step`.
```
Comment (craigtaverner) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ListFunctionsTest.scala]: Spelling mistake on 'integer'

Comment (sherfert) [inline cypher/cypher-docs/src/test/scala/org/neo4j/cypher/docgen/ListFunctionsTest.scala]:
```suggestion
          |To create ranges with decreasing integer values, use a negative value `step`.
```

**Review by davidoliverSP2 — APPROVED** [review]
thanks @sherfert 👍 

is this PR still waiting on any further reviews?

**Comment by sherfert** [comment]
@davidoliverSP2 , it's ready to go in. Since it is 4.0, I guess you will (forward-)merge it?

**Pull request merged**